client/src/app/site/projector/services/current-list-of-speakers.service.ts

```diff
@@ -130,21 +153,11 @@ export class CurrentListOfSpeakersService {
      * @returns The view list of speakers or null, if there is no such projector element.
      */
     private getCurrentListOfSpeakersForProjector(projector: ViewProjector): ViewListOfSpeakers | null {
-        throw new Error('TODO');
-        /*const nonStableElements = projector.elements.filter(element => !element.stable);
-        for (const nonStableElement of nonStableElements) {
-            const identifiableNonStableElement = this.slideManager.getIdentifiableProjectorElement(nonStableElement);
-            try {
-                const viewModel = this.projectorService.getViewModelFromIdentifiableProjectorElement(
-                    identifiableNonStableElement
-                );
-                if (isBaseViewModelWithListOfSpeakers(viewModel)) {
-                    return viewModel.listOfSpeakers;
-                }
-            } catch (e) {
-                // make TypeScript silent.
+        for (const projection of projector.current_projections) {
```

Comment:
Nice, having shorter code as in OS3 is an indication, that we did something right with the new concept ;)